bakerydemo/templates/base/preview/static_embed_block.html

```diff
@@ -0,0 +1,6 @@
+{% extends "wagtailcore/shared/block_preview.html" %}
+{% load static %}
+
+{% block content %}
+    <img class="static-block-preview" src="{% static "img/embed-block-preview.png" %}" alt="Preview of an embedded video" />
+{% endblock %}
```

Comment:
I thought it'd be nice to showcase how a static image preview could work with the current implementation.

Reply:
I agree, but I’d rather we demo-ed with an SVG so it’s less file to load and will be easier to dark-theme-ify